contrib/migrate/gogs_migrate.sh

```diff
@@ -191,7 +191,7 @@ cp -R data "$gitea_path"
 #mv $gitea_path/conf $gitea_path/options
 cd "$gitea_path"
 mv "custom/conf/app.ini" "custom/conf/gogs_app.ini"
-url="https://raw.githubusercontent.com/go-gitea/gitea/v$gitea_version/conf/app.ini"
+url="https://raw.githubusercontent.com/go-gitea/gitea/v$gitea_version/custom.sample/app.ini"
```

Comment:
This looks suspicious. BTW, I see gitea_version is hardcoded to 1.0.1 so this would basically not work as the 1.0.1 tag would not have custom.sample...